setup.py

```diff
@@ -11,7 +11,7 @@
 setup(
 
         name='trustymail',
-        version='0.0.1',
+        version=__version__,
```

Comment:
It looks like `setup.py` is missing the `from trustymail import __version__` call.

Reply:
@IanLee1521 Are you saying there is one bug left to iron out?  I think I've lost the thread here...it looks like all the requested changes have been made.

Reply:
correct, currently there is no import in that file to get that value... (Only in the other PR).